js/modules/k6/grpc/client.go

```diff
@@ -367,6 +379,24 @@ func (c *Client) parseConnectParams(raw map[string]interface{}) (connectParams,
 			if !ok {
 				return params, fmt.Errorf("invalid reflect value: '%#v', it needs to be boolean", v)
 			}
+		case "maxReceiveSize":
+			var ok bool
+			params.MaxReceiveSize, ok = v.(int64)
+			if !ok {
+				return params, fmt.Errorf("invalid maxReceiveSize value: '%#v', it needs to be an int", v)
```

Comment:
```suggestion
				return params, fmt.Errorf("invalid maxReceiveSize value: '%#v', it needs to be an integer", v)
```

Reply:
Just changed every "int" for "integer" (actually feels better when reading the error).

If any more changes are needed after the next review, I will do them in a separate commit.